pgjdbc/src/main/java/org/postgresql/ds/common/BaseDataSource.java

```diff
@@ -1076,6 +1076,15 @@ public String getUrl() {
   }
 
   /**
+   * Generates a {@link DriverManager} URL from the other properties supplied.
+   *
+   * @return {@link DriverManager} URL from the other properties supplied
+   */
+  public String getURL() {
```

Comment:
Can you add a test for getURL? 

Also check that the values of server, database, user, etc... are correctly set.

Reply:
I've added tests for getURL and setURL verifying provided parameters.